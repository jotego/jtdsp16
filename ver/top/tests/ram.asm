r0=0x100
r1=0x23
r2=0xbabe
r3=0xa055
j=3
# Test ++ and --
*r0++=r1
*r0++=r2
*r0=r3
r1=*r0--
r2=*r0--
r3=*r0--
end:
goto end
