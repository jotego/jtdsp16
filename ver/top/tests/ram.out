r0=0xfeff
r1=0xa055
r2=0xbabe
r3=0x0023
rb=0x0000
re=0x0000
j =0x0003
k =0x0000
pr=0x0000
pt=0x0000
i =0x0000
a0=0x000000000
a1=0x000000000
